django/db/models/functions/__init__.py

```diff
@@ -10,8 +10,9 @@
     Mod, Pi, Power, Radians, Round, Sin, Sqrt, Tan,
 )
 from .text import (
-    Chr, Concat, ConcatPair, Left, Length, Lower, LPad, LTrim, Ord, Repeat,
-    Replace, Reverse, Right, RPad, RTrim, StrIndex, Substr, Trim, Upper,
+    MD5, Chr, Concat, ConcatPair, Left, Length, Lower, LPad, LTrim, Ord,
```

Comment:
I don't know why isort sorts M before C. Is there a special rule for names in all caps?

Reply:
`isort` treats `MD5` as a constant and sorts imports for CONSTANTS, Classes, and modules/functions (see [`order_by_type` setting](https://github.com/timothycrosley/isort/wiki/isort-Settings#full-reference-of-isort-settings)).